addons/mail/tests/discuss/test_discuss_attachment_controller.py

```diff
@@ -79,4 +79,4 @@ def test_first_page_access_of_mail_attachment_pdf(self):
             ownership_token = attachment._get_ownership_token()
             url = f'/mail/attachment/pdf_first_page/{attachment.id}?access_token={ownership_token}'
             response = self.url_open(url)
-            self.assertEqual(response.status_code, 415)
+            self.assertIn(response.status_code, [415, 200])
```

Comment:
I'd at least specify that this depends on whether pypdf and the crypto lib are installed or not with some comment